index.bs

```diff
@@ -386,14 +404,15 @@ a different [=origin=], by incorporating the [=origin=] in its responses. Specif
 the full [=origin=] of the requester is included, and signed over, in the [=attestation object=] produced when a new credential
 is created as well as in all assertions produced by WebAuthn credentials.
 
-Additionally, to maintain user privacy and prevent malicious [=[RPS]=] from probing for the presence of credentials belonging to
-other [RPS], each credential is also associated with a Relying Party Identifier, or RP ID. This RP ID is provided by the client
-to the authenticator for all operations, and the authenticator ensures that credentials created by a [=[RP]=] can only be used
-in operations requested by the same RP ID. Separating the [=origin=] from the RP ID in this way allows the API to be used in
-cases where a single [=[RP]=] maintains multiple [=origins=].
+Additionally, to maintain user privacy and prevent malicious [=[RPS]=] from probing for the presence of [=public key
+credentials=] belonging to other [=[RPS]=], each [=public key credential|credential=] is also associated with a [=Relying Party
+Identifier=], or [=RP ID=]. This [=RP ID=] is provided by the client to the [=authenticator=] for all operations, and the
```

Comment:
this is a topic for a discussion of our overall so-far ad-hoc spec source editing conventions and techniques (e.g. the use of regex for search and replace)